lib/src/component_declaration/component_base.dart

```diff
@@ -675,6 +675,7 @@ abstract class UiProps extends MapBase
   }
 
   ReactComponentFactoryProxy get componentFactory;
+  set componentFactory(ReactComponentFactoryProxy v) => throw new UnsupportedError('No');
```

Comment:
This could use a better error message 😜

Reply:
Ohhhhhh i seee `Uncaught Unsupported operation: No` isn't good enough for you?